### ISSUE TITLE
Player colours

### ISSUE THREAD

**Issue by daggeg** [Vocaluxe/Vocaluxe#681]
Is there anyway you can implement the choice of players' colours in-game? Every time I update the game to the latest nightly I replace the datafiles containing the players' colours to the colour scheme I prefer.

**Comment by flokuep** [comment]
Nope, that's not possible at the moment. Are there reasons why you're changing that? Is that something you change once, or depending on the player singing?

As a workaround, I suggest to copy one skin xml file (the one you like at most), and name it different, so that it will resist if you copy-over new nightly builds. Actually skin changes could break your custom skin, but you should see that fast and can readjust it?
